packages/elements/src/slider/elements/slider.ts

```diff
@@ -141,6 +141,10 @@ export class Slider extends ControlElement {
   private fromPreviousInput = ''; // dynamically accessed
   private toPreviousInput = ''; // dynamically accessed
 
+  /** Aria label for the 'to' and 'from' slider, resolved based on locale. */
+  private toAriaLabel = 'to';
+  private fromAriaLabel = 'from';
```

Comment:
They should initailly be empty stringas they are to be resolved based on locale?
```suggestion
  private toAriaLabel = '';
  private fromAriaLabel = '';
```